rdkit/Chem/Features/FeatDirUtilsRD.py

```diff
@@ -320,6 +295,8 @@ def GetDonor2FeatVects(conf, featAtoms, scale=1.5):
       bvec += cpt
       res.append((cpt, bvec))
     return tuple(res), 'linear'
+  
+  return None
```

Comment:
This looks like something pylint is complaining about. Did you use pylint to check the code?

Reply:
Unfortunately, I did not use pylint to check, but since I saw couple of empty return (`return `) and probably unclear result to handle. This is just to make thing clear.